### DISCUSSION TITLE
Per session remember user settings

### DISCUSSION THREAD

**Discussion by mylife4aiurr** [sn4k3/UVtools#195] [Ideas]
When using tools like exposure finder or elephants foot compensation I would really like if uvtools remembered the options I choose while the program is still in memory. Every time I open these tools they default to default. This is frustrating, its like starting from scratch every time. 

Im aware one can setup a profile and have that be the new default, but thats after you find a set of parameters you like and want to use all the time. 

Thats not what I'm after. When users dont understand what certain options do they may try changing a parameter or two and see how the output looks. But once they re-enter the tool the last parameters they tried are gone.

**Comment by sn4k3** [comment]
Have you tried the Ctrl + Shift + Z? I use it all the time for that purpose

**Reply by mylife4aiurr** [reply]
Ok thanks I'll use that going forward.
I still think this functionality would help users be more efficient. I'm not sure all users going forward know the short cut keys, like programmers.

**Reply by sn4k3** [reply]
Yes i can enable a option like that in a future release, let-me rework this
